sdk/nodejs/cmd/dynamic-provider/index.ts

```diff
@@ -187,7 +188,8 @@ async function createRPC(call: any, callback: any): Promise<void> {
 
         callback(undefined, resp);
     } catch (e) {
-        return callback(grpcResponseFromError(e));
+        const response = grpcResponseFromError(e);
+        return callback(/*err:*/ response, /*value:*/ null, /*metadata:*/ response.metadata);
```

Comment:
grpc-js wants to be a drop-in replacement for grpc.  However, their behavior is different here (which I reported to them here: https://github.com/grpc/grpc-node/issues/1236).  This works around their compat break here.